.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run pre-commit
```

Comment:
probably worth upgrading husky at some point in a separate PR

Reply:
Husky is already upgraded to the latest version as part of this PR. That's why this file is needed.

Reply:
ah ignore me, this _is_ the new one!